code/modules/paperwork/paperplane.dm

```diff
@@ -30,6 +30,8 @@
 		newPaper.forceMove(src)
 	else
 		internalPaper = new(src)
+	if(internalPaper.icon_state == "cpaper" || internalPaper.icon_state == "cpaper_words")
```

Comment:
Why not check `iscopy`, which seems to control this icon state?

Reply:
As `internalPaper` is not guaranteed to be a `paper/Carbon` and thus might not have `is copy` this would not compile.
I tried around with `istype()` a bit, but could not find a more elegant solution.
As I'm new to DM, I'd be happy to see a better solution.

Reply:
Good enough for me.